ipmi-config/ipmi-config-validate.h

```diff
@@ -114,6 +114,11 @@ ipmi_config_validate_t ip_address_source_number_validate (ipmi_config_state_data
                                                           const char *key_name,
                                                           const char *value);
 
+ipmi_config_validate_t ipv6_ipv4_addressing_enables_validate (ipmi_config_state_data_t *state_data,
+							      const char *section_name,
+							      const char *key_name,
+							      const char *value);
+
```

Comment:
the ipmi-config-validate.[ch] files are more for "common" validation functions that cross files.  I imagine this one is unique to the lan_conf_ipv6 file you've created, so you can just move in there (although I could be wrong, do you see it being used elsewhere??)